cmake/Install.cmake

```diff
@@ -241,7 +244,6 @@ endif()
 
 if( APPLE )
   set(CPACK_PACKAGE_DEFAULT_LOCATION "/Applications")
-  set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
```

Comment:
Is this variable not picked up by anyone else later in the IFW build?  It looks to be unused, but I'm still really nervous about touching this since it works perfectly fine right now.

Reply:
It's already defined on line 10: https://github.com/NREL/EnergyPlus/pull/7313/files/cfbb86e3d7fd51e3c80dcf5102a1ab6937fd8121#diff-41362486261ba172ec8a6127c1785ccfR10

Only difference is that it's defined with the build sha too:

```
set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}-${CPACK_PACKAGE_VERSION_BUILD}")
```

I'll revert that too.